docs/arm64.md

```diff
@@ -1,17 +1,21 @@
 # How to build and test Cloud Hypervisor on AArch64
 
-This document introduces how to build and test Cloud Hypervisor on AArch64
-servers. Currently Cloud Hypervisor cannot be tested on Raspberry PI. Because
-on AArch64, Cloud Hypervisor requires GICv3-ITS device for PCIe MSI interrupt
-handling. But GICv3-ITS has not been equipped on any Raspberry PI product so
-far.
-
-Now Cloud Hypervisor supports 2 ways of booting on AArch64: UEFI booting and
-direct-kernel booting. The document covers both of the ways.
+This document introduces how to build and test Cloud Hypervisor on AArch64.
+Currently, Cloud Hypervisor supports 2 methods of booting on AArch64: UEFI
+booting and direct-kernel booting. The document covers both methods.
 
 All the steps are based on Ubuntu. We use the Ubuntu cloud image for guest VM
 disk.
 
+## Hardware requirements
+
+- AArch64 servers (recommended) or development boards equipped with the
+GICv3-ITS interrupt controller.
```

Comment:
I am using the iMX8M EVK - to the best of my knowledge it has a GICv3 but no ITS. In my dmesg I see:
```[    0.000000] ITS: No ITS available, not enabling LPIs```

There was no problem booting Linux VMs with cloud-hypervisor or QEMU. However, booting a Windows VM on QEMU required me to disable its (`its=off`). Cannot say anything about CLH+Win VM.

Reply:
@ms-mahuber You are right, creating a virtual ITS doesn't require physical ITS. GICv3 is enough. This is described in kernel doc: https://www.kernel.org/doc/html/latest/virt/kvm/devices/arm-vgic-its.html

Thank you for catching this.

Reply:
Thanks @ms-mahuber @michael2012z , and done.